CMakeLists.txt

```diff
@@ -57,11 +57,70 @@ endif()
 
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/cmake)
 
+# To configure mbedtls
+# todo make the configuration better
+set(MBEDTLS_CONFIG_FILE "mbedtls_config.h")
+include_directories(${CMAKE_SOURCE_DIR}/lib/include)
```

Comment:
yuk`include_directories`

Reply:
@lurch suggested it https://github.com/raspberrypi/picotool/pull/207#discussion_r2003612398 - I can go back to what it was before if you prefer

It had to be a bit yuck, as it needs to be applied to the mbedtls build without modifying it's CMake files

Reply:
oh ok. maybe a comment then